collector/cluster-insight-controller.json

```diff
@@ -0,0 +1,61 @@
+{
+    "kind": "ReplicationController",
+    "apiVersion": "v1beta3",
+    "metadata": {
+	"name": "cluster-insight-controller",
+	"labels": {
+	    "name": "cluster-insight"
+	}
+    },
+    "spec": {
+	"replicas": 1,
```

Comment:
can you add comments to this JSON file?
if you can, please say that it is only the default number of replicas, and that the kubctl command used to deploy the cluster insight should set the correct number of replicas (one in each minion nodes).